application/frontend/web/load-app.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+require(__DIR__ . '/../../vendor/autoload.php'); //TODO: why is this required for the web app but not the tests?
```

Comment:
not PHPUnit, but Behat.  I assume the same applies though.  I'd like to turn off that autoloader for the tests if possible then, is that controlled through composer somehow?